x/posts/legacy/v0.8.0/migrate_test.go

```diff
@@ -0,0 +1,206 @@
+				LastEdited:     time.Time{},
+				Creator:        parentPostCreator,
+				Medias:         []v040.PostMedia{{URI: "https://uri.com", MimeType: "text/plain"}},
+			},
+			{
+				PostID:         postID,
+				ParentID:       parentID,
+				Message:        "Message",
+				AllowsComments: true,
+				Subspace:       "4e188d9c17150037d5199bbdb91ae1eb2a78a15aca04cb35530cccb81494b36e",
+				OptionalData:   map[string]string{},
+				Created:        postCreationTime,
+				LastEdited:     time.Time{},
+				Creator:        postCreator,
+				Medias:         []v040.PostMedia{{URI: "https://uri.com", MimeType: "text/plain"}},
+			},
+		},
+		UsersPollAnswers: map[string][]v040.UserAnswer{string(postID): {v040.UserAnswer{
+			Answers: []v040.AnswerID{1, 2},
+			User:    postCreator,
+		}}},
+		PostReactions: map[string][]v060.PostReaction{string(postID): {
+			v060.PostReaction{
+				Owner:     postCreator,
+				Shortcode: ":fire:",
+				Value:     "🔥",
+			},
+			v060.PostReaction{
+				Owner:     postCreator,
+				Shortcode: ":my_house:",
+				Value:     "https://myHouse.png",
+			},
+		}},
+		RegisteredReactions: []v040.Reaction{
+			{
+				ShortCode: ":my_house:",
+				Value:     "https://myHouse.png",
+				Subspace:  subspace,
+				Creator:   postCreator,
+			},
+		},
+	}
+
+	migrated := v080.Migrate(v060GenState)
+
+	// Check for posts
+	require.Len(t, migrated.Posts, len(expected.Posts))
+	for index, post := range migrated.Posts {
+		require.Equal(t, expected.Posts[index].PostID, post.PostID)
+		require.Equal(t, expected.Posts[index].ParentID, post.ParentID)
+	}
+
+	// Check for users poll answers
+	require.Len(t, migrated.UsersPollAnswers, len(expected.UsersPollAnswers))
+	for key := range expected.UsersPollAnswers {
+		require.Equal(t, expected.UsersPollAnswers[key], migrated.UsersPollAnswers[key])
+	}
+
+	// Check for posts reactions
+	require.Len(t, migrated.PostReactions, len(expected.PostReactions))
+	for key := range expected.PostReactions {
+		require.Equal(t, expected.PostReactions[key], migrated.PostReactions[key])
+	}
+
+	require.Len(t, migrated.RegisteredReactions, len(expected.RegisteredReactions))
+	for index, reaction := range migrated.RegisteredReactions {
+		require.Equal(t, expected.RegisteredReactions[index], reaction)
+	}
+
+}
+
+func TestMigrate060(t *testing.T) {
+	config := sdk.GetConfig()
+	config.SetBech32PrefixForAccount("desmos", "desmos"+sdk.PrefixPublic)
+	config.Seal()
+
+	content, err := ioutil.ReadFile("v060state.json")
+	require.NoError(t, err)
+
+	var v060state v060.GenesisState
+	err = json.Unmarshal(content, &v060state)
+	require.NoError(t, err)
+
+	v080state := v080.Migrate(v060state)
+	for _, reaction := range v080state.RegisteredReactions {
+		// Make sure each reaction shortcode does not represent an emoji
+		_, err := emoji.LookupEmojiByCode(reaction.ShortCode)
+		require.Error(t, err)
+
+		// Make sure no reaction value is an emoji
+		_, err = emoji.LookupEmoji(reaction.Value)
+		require.Error(t, err)
+	}
+
```

Comment:
I think we should check if the params for the `posts` and `profiles` modules have been set correctly as well